deploy/olm-catalog/ibm-common-service-operator/3.4.0/ibm-common-service-operator.v3.4.0.clusterserviceversion.yaml

```diff
@@ -37,50 +37,39 @@ spec:
       kind: CommonService
       name: commonservices.operator.ibm.com
       version: v3
-  description: |-
-    This operator is a bridge to connect CloudPaks and ODLM/Common Services, and it can be also installed in standalone mode.
+  description: The `ibm-common-service-operator` is a bridge to connect IBM Cloud Paks and Operand Deployment Lifecycle Manager (ODLM) with IBM Cloud Platform Common Services. You can also install the `ibm-common-service-operator` in stand-alone mode.
```

Comment:
you should use block indicator here(`description: |-`), see: https://yaml.org/spec/1.2/spec.html#id2794534